src/lib.rs

```diff
@@ -317,6 +334,11 @@ pub fn run(mut config: Config) {
         compressor.stats.state_groups_changed
     );
 
+    if ratio > 1.0 {
+        println!("This compression would not remove any rows. Aborting.");
```

Comment:
```suggestion
        println!("This compression would not remove any rows. Exiting.");
```

`abort` typically means something has gone wrong, and that we're going to return with an error code.